pkg/backend/remote/remote.go

```diff
@@ -20,8 +21,10 @@ import (
 )
 
 var (
+	ErrPingTimeout = errors.New("Ping timeout")
+
 	pingInveral   = 2 * time.Second
-	timeout       = 30 * time.Second
+	pingTimeout   = 2 * time.Second
```

Comment:
2 seconds is quite a small ping timeout. I am not sure if we should set a timeout so small in the data connection layer.

Reply:
I'll change it to eight seconds which is what it previously was in dataconn.Client.